source/uwp/Renderer/lib/AdaptiveCard.cpp

```diff
@@ -303,12 +293,10 @@ AdaptiveNamespaceStart
         adaptiveCard->SetSpeak(HStringToUTF8(m_speak.Get()));
         adaptiveCard->SetLanguage(HStringToUTF8(m_language.Get()));
 
-        if (m_backgroundImage != nullptr)
+        if (m_backgroundImageUri != NULL)
```

Comment:
Now that this is just a string, you can simplify this using a call to the version of HStringToUTF8 that doesn't fail on null values:

adaptiveCard->SetBackgroundImage(HStringToUTF8(m_backgroundImageUri.Get()));